webapp/templates/main.html

```diff
@@ -42,7 +42,11 @@ <h4>Random passenger of the Day:</h4>
     </div>
 
     <!-- For Erika's R Analysis -->
-    <div class="container-fluid">
+    <div class="col-md-4">
+        <h2>Statistical Analysis Using R</h2>
+        <p>The Chi-Squared Test for independence was utilized to for this analysis. It compares two variables in a contingency table to see if they are related. In a more general sense, it tests to see whether distributions of categorical variables differ from each other.  We tested the variables of gender, class, embarkment, and age against the chance of survival.</p> 
+        <p align = "center"><iframe src="/C:\Users\Meg Weightman\MSU\finalProject\children-of-jupyter\Resources\R Resources\analysis-output.txt" width="500" height="300" frameBorder="0"></iframe></p>
```

Comment:
We'll need to change this to a relative pathlink or perma link to make it accessible from the repo, but I will take care of this later today.